docs/integration/solrcloud.md

```diff
@@ -1,9 +1,112 @@
+    --agent-type javaagent  \
+    --infra-token <your-infra-token-goes-here>
+```
+
+The above command will setup your SolrCloud Monitoring Agent in the *in-process* mode, to have it running in the *standalone* mode instead of running the above command, run the one shown below:
+```bash
+sudo bash /opt/spm/bin/setup-sematext  \
+    --monitoring-token <your-monitoring-token-goes-here>   \
+    --app-type solr  \
+    --agent-type standalone  \
+    --infra-token <your-infra-token-goes-here>  \
+    --jmx-params '-Dspm.remote.jmx.url=localhost:3000'
+```
+
+Keep in mind that your need to provide the Monitoring token and Infra token. They are both provided in the [installation instructions](https://apps.sematext.com/ui/howto/SolrCloud/overview) for your SolrCloud App. 
+
+Finally, the last thing that needs to be done is adjusting the *solr.in.sh* file and add the following section:
+```bash
+SOLR_OPTS="$SOLR_OPTS -Dcom.sun.management.jmxremote -javaagent:/opt/spm/spm-monitor/lib/spm-monitor-generic.jar=<your-monitoring-token-goes-here>::default"
+```
+
+Or if you would like to run the Solr Monitoring Agent in the *standalone* mode add the following section to the ```solr.in.sh``` file:
+```bash
+SOLR_OPTS="$SOLR_OPTS -Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.port=3000 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false"
+```
+
+Make sure that tag ```<jmx />``` is enabled in your ```solrconfig.xml``` file.
+
+**You need to restart your SolrCloud node after the above changes.**
+
+## Collected Metrics
+
+The Sematext SolrCloud monitoring agent collects the following metrics.
+
+### Operating System
+
+- CPU usage
+- CPU load
+- Memory usage 
+- Swap usage
+- Disk space used
+- I/O Reads and Writes
+- Network traffic
+
+![](https://sematext.com/wp-content/uploads/2019/05/d_solr_cpu_details.png)
+
+### Java Virtual Machine
+
+- Grabage collectors time and count
+- JVM pool size and utilization
+- Threads and daemon threads
+- Files opened by the JVM
+
+![](https://sematext.com/wp-content/uploads/2019/05/d_solr_jvm_pool.png)
+
+### Solr
+
+- Requests rate and latency 
+- Solr index stats and file system stats
+- Added and pending documents
+- Deletes by id and queries
+- Filter cache statistics
+- Document cache statistics
+- Query result cache statistics
+- Per segment filter cache statistics
+- Commit events 
+- Warmup times
+
+![](https://sematext.com/wp-content/uploads/2019/05/d_solr_requests.png)
+
+## Troubleshooting
+
+If you are having issues with Sematext Monitoring, i.e. not seeing SolrCloud metrics, you can create diagnostics package on affected machines where SolrCloud Monitoring Agent installed by running:
+
+```bash
+sudo bash /opt/spm/bin/spm-client-diagnostics.sh
+```
+
+The resulting package will contain all relevant info needed for our investigation. You can send it, along with short description of your problem, to support@sematext.com or contact us in the chat.
```

Comment:
The resulting package will contain all relevant info needed for our investigation. You can send it, along with **a** * short description of your problem, to support@sematext.com or **contact us through the chat in the bottom right.** *

Reply:
Adjusted.